src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -26,9 +20,21 @@ public Window()
 			CoreWindow = new CoreWindow();
 		}
 
-		private void InternalActivate()
+		public int SystemUiVisibility { get; internal set; }
+
+		public bool IsNavigationBarVisible
 		{
+			get
+			{
+				var decorView = (ContextHelper.Current as Activity)?.Window?.DecorView;
 
+				return (SystemUiVisibility & (int)SystemUiFlags.HideNavigation) == 0
+					&& (decorView == null || ((int)decorView.SystemUiVisibility & (int)SystemUiFlags.HideNavigation) == 0);
+			}
+		}
+
+		private void InternalActivate()
```

Comment:
?

Reply:
Existing code (Line 29 on the original version)

Reply:
Ok, well, remove it ;)

Reply:
Done